nitrous.readme.md

```diff
@@ -0,0 +1,68 @@
+# Nitrous Brunch Quickstart
```

Comment:
That does sound better, makes a lot of sense. Quickstarts for specific Brunch skeleton repos can be listed and linked on there too - as suggested by [Colin](https://github.com/colinbate) below. If you create the new `brunch-nitrous` repo I'll submit another pull request to it with these proposed Quickstart files (and some amendments). 

That sound good? Or is there a better way?